app/assets/javascripts/discourse/tests/unit/lib/plugin-api-test.js

```diff
@@ -69,23 +80,44 @@ module("Unit | Utility | plugin-api", function (hooks) {
       }
     }
 
-    getOwner(this).register("class-test-thingy:main", new ClassTestThingy(), {
+    getOwner(this).register("class-test-thingy:main", ClassTestThingy, {
       instantiate: false,
```

Comment:
We should probably instantiate now that it's a class?

Reply:
If I set `instantiate: true` it raises an error. The Ember registry only seems to be capable of instantiating things which have a `create()` method. (plus, I guess it has special handling for Glimmer components).

I don't think it really matters for this test - the important thing is that the class (not the instance) is in the registry. It's totally fine for us to do the initialization ourselves after the `lookup()`

Reply:
Actually ignore me - I'm getting confused between the `instantiate` option and the `singleton` option. We don't need either for the purpose of this test. Removed in latest commit :+1: